Hw1

ZhongjunHu opened the pull request:
I will polish and add analysis later!

malecki committed:
Merge pull request #3 from mm3940/assignment1

Assignment1

ZhongjunHu committed:
Merge remote-tracking branch 'upstream/gh-pages' into gh-pages

Conflicts:
	.DS_Store

ZhongjunHu committed:
commit

ZhongjunHu committed:
initial commit

Merge remote-tracking branch 'upstream/questionnaire' into questionnaire

ZhongjunHu committed:
initial commit

ZhongjunHu committed:
bu老师教学

ZhongjunHu committed:
add sth

ZhongjunHu committed:
comment

ZhongjunHu committed:
222

ZhongjunHu committed:
final

ZhongjunHu committed:
Merge remote-tracking branch 'upstream/gh-pages' into hw1

Conflicts:
	.gitignore

malecki commented:
sounds good. thanks!

On Thu, Sep 18, 2014 at 10:38 PM, ZhongjunHu notifications@github.com
wrote:

> ## I will polish and add analysis later!
> 
> You can merge this Pull Request by running
> 
>   git pull https://github.com/ZhongjunHu/qmssviz hw1
> 
> Or view, comment on, or merge it at:
> 
>   https://github.com/malecki/qmssviz/pull/29
> Commit Summary
> - Merge pull request #3 from mm3940/assignment1
> - Merge remote-tracking branch 'upstream/gh-pages' into gh-pages
> - commit
> - initial commit
> - initial commit
> - bu老师教学
> - add sth
> - comment
> - 222
> - final
> 
> File Changes
> - _M_ .gitignore
>   https://github.com/malecki/qmssviz/pull/29/files#diff-0 (1)
> - _A_ hw/hw1.md
>   https://github.com/malecki/qmssviz/pull/29/files#diff-1 (0)
> - _A_ hw1/_posts/2014-09-18-Zhongjun-Hu.Rmd
>   https://github.com/malecki/qmssviz/pull/29/files#diff-2 (42)
> - _A_ hw1/_posts/2014-09-18-Zhongjun-Hu.html
>   https://github.com/malecki/qmssviz/pull/29/files#diff-3 (128)
> - _A_ hw1/_posts/hw1 huzhongjun.Rmd
>   https://github.com/malecki/qmssviz/pull/29/files#diff-4 (22)
> - _A_ labs/questionnaire/responses-2014-09-05.csv
>   https://github.com/malecki/qmssviz/pull/29/files#diff-5 (56)
> - _A_ labs/questionnaire/transform.Rmd
>   https://github.com/malecki/qmssviz/pull/29/files#diff-6 (55)
> - _A_ labs/questionnaire/transform.html
>   https://github.com/malecki/qmssviz/pull/29/files#diff-7 (85)
> 
> Patch Links:
> - https://github.com/malecki/qmssviz/pull/29.patch
> - https://github.com/malecki/qmssviz/pull/29.diff
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/malecki/qmssviz/pull/29.

ZhongjunHu committed:
jj

Merge remote-tracking branch 'upstream/gh-pages' into hw1

Conflicts:
	Lab1/Lab1_final.html

ZhongjunHu committed:
elliot assign1

ZhongjunHu committed:
revise1

ZhongjunHu committed:
revise2

ZhongjunHu committed:
rvs3

ZhongjunHu committed:
revise4

ZhongjunHu committed:
revise 5
